source/mainnet/smart-contracts/tutorials/voting/voting-sc.rst

```diff
@@ -86,4 +85,20 @@ And if the end time has passed, an error occurs.
 Initializing
 ------------
 
-The election is open from the point in time that this smart contract is initialized until the `end_time`.
+The election is open from the point in time that this smart contract is initialized until the deadline.
+
+Performance considerations
+--------------------------
+
+An important aspect of the voting smart contract to highlight is the way that votes are tallied. Note how two ``HashMap``s are used: one for which accounts have voted for which option, and one for how many votes each option has.
+However, if you think about it, only the first ``HashMap``, the ``ballots`` field is necessary. The vote counts can always be determined from those. So why do it in this way?
```

Comment:
The cursive styling does not stop after ``HashMap``.

Reply:
Huh, I guess it doesn't like the s directly after the code/monospace (not cursive) styling? Will see if I can fix that.